backport_packages/setup_backport_packages.py

```diff
@@ -155,6 +156,11 @@ def pure_airflow_models_filter(node: LN, capture: Capture, filename: Filename) -
         """Check if select is exactly [airflow, . , models]"""
         return len([ch for ch in node.children[1].leaves()]) == 3
 
+    def remove_super_init_call(node: LN, capture: Capture, filename: Filename) -> None:
+        for ch in node.post_order():
+            if hasattr(ch, "value") and ch.value == "super":
```

Comment:
can we have something else than super().init() removed here? I do not see check that it is actually an init() method?

Reply:
I've added an additional check ✅